homeassistant/components/august/const.py

```diff
@@ -26,12 +26,16 @@
 OPERATION_METHOD_AUTORELOCK = "autorelock"
 OPERATION_METHOD_REMOTE = "remote"
 OPERATION_METHOD_KEYPAD = "keypad"
+OPERATION_METHOD_MANUAL = "manual"
+OPERATION_METHOD_TAG = "tag"
 OPERATION_METHOD_MOBILE_DEVICE = "mobile"
 
 ATTR_OPERATION_AUTORELOCK = "autorelock"
```

Comment:
I was planning on deprecating the true/false attributes since you can figure it out fully from the operation one.  

I would say we probably shouldn't add new ones but it won't cause a lot of database churn since they are mutually exclusive and it's best to do away with them in a future PR when we can give 6 months for users to adapt

Reply:
I agree that the operation should be enough.  The true/false sounds more like debug info, but for this PR I just followed the other attributes.